python/mypy.ini

```
@@ -2,5 +2,5 @@
 strict_optional = False
 ignore_missing_imports = True
 disallow_untyped_decorators = False
-follow_imports = "silent"
+follow_imports = silent
```

We should use the recommended mypy settings

https://www.notion.so/fulcrumgenomics/Python-2ecc3f8900e143f0bbcfee3f12b50809?pvs=4#9bc73f34e57d45ccbdeda5afdb76458e

```suggestion
strict_optional             = false
check_untyped_defs          = true
disallow_incomplete_defs    = true
disallow_untyped_calls      = true
disallow_untyped_decorators = true
disallow_untyped_defs       = true
no_implicit_optional        = true
warn_no_return              = true
warn_redundant_casts        = true
warn_return_any             = true
warn_unreachable            = true
warn_unused_configs         = true
warn_unused_ignores         = true
```